sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -116,8 +125,21 @@ private void startTracing(final @NonNull Activity activity) {
       stopPreviousTransactions();
 
       // we can only bind to the scope if there's no running transaction
-      final ITransaction transaction =
-          hub.startTransaction(getActivityName(activity), "navigation");
+      ITransaction transaction;
+      final String activityName = getActivityName(activity);
+
+      final Date appStartTime = AppStartState.getInstance().getAppStartTime();
+
+      // in case appStartTime isn't available, we don't create a span for it.
+      if (firstActivityCreated || appStartTime == null) {
+        transaction = hub.startTransaction(activityName, NAV_OP);
+      } else {
+        // start transaction with app start timestamp
+        transaction = hub.startTransaction(activityName, NAV_OP, appStartTime);
+        // start specific span for app start
+        // TODO: add description cold/warm
+        appStartSpan = transaction.startChild("app.start", appStartTime);
```

Comment:
`h`: We still need to align on Cocoa and Android with operation name and span operations for this.

Reply:
`app.start` it is 😁